wasm/src/logging.rs

```diff
@@ -0,0 +1,168 @@
+use core::sync::atomic::{AtomicU8, Ordering};
+
+use super::*;
+
+// NOTE: `u8::MAX` is a sentinel value for an undefined log level
+static MAX_LOG_LEVEL: MaxLogLevel = MaxLogLevel(AtomicU8::new(u8::MAX));
+
+/// Struct which holds a valid [`LogLevel`] stored as an integer
+struct MaxLogLevel(AtomicU8);
+
+/// Log level supported by the host
+// NOTE: This struct must be exact duplicate of `config::logger::Level`
```

Comment:
But we can't just import it, right?

Reply:
I don't want to be dealing with `iroha_logger` crate as yet another dependency in `wasm` and making that crate `no_std` and FFI compatible just for a single enum. I tried moving this enum into `data_model` so I can share it but it just gave me headaches